README.md

```diff
@@ -616,14 +616,31 @@ midtrans.DefaultGoHttpClient = &http.Client{
 ## 4. Handling Error
 When using function that result in Midtrans API call e.g: c.ChargeTransaction(...) or s.CreateTransaction(...) there's a chance it may throw error (Midtrans [Error object](/error.go)), the error object will contains below properties that can be used as information to your error handling logic:
 ```go
-    _, err = c.chargeTransaction(param);
+    _, err = c.chargeTransaction(param)
     if err != nil {
         msg := err.GetMessage()                // general message error
         stsCode := err.GetStatusCode()         // HTTP status code e.g: 400, 401, etc.
         rawApiRes := err.GetRawApiResponse()   // raw Go HTTP response object
         rawErr := err.GetRawError()            // raw Go err object
     }
```

Comment:
@Xaxxis Why not demonstrate calling `err.Error()` & `err.Unwrap()` here? as a more modern way compared to traditional `msg := err.GetMessage()` & `rawErr := err.GetRawError()` 
please add.
Other than that looks good. After doing above point please merge & release, no need to wait my review again.